ocaml/idl/datamodel_pool.ml

```diff
@@ -75,6 +75,21 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  (* This is a map of uuid -> cert_blob *)
+  let pool_certs = Map (String, String)
+
+  let pre_join_exchange_certificates = call
+      ~name:"pre_join_exchange_certificates"
+      ~in_oss_since:None
+      ~in_product_since:rel_next
+      ~params:[String, "uuid", "The uuid of the joiner";
+               String, "certificate", "The contents of the joiner's certificate";
```

Comment:
I assume this is only the public part of the certificate. Mention this?

Reply:
Technically the private key is not part of the certificate, the format used to stored the certificate along the private key is PKCS #12. Note that the certificate records never contain any information about private keys.